app/main.py

```diff
@@ -46,6 +46,7 @@ def create_tables(engine, psql_environment):
     search.router,
     telegram.router,
     whatsapp.router,
+    fourOfour.router,
```

Comment:
Please keep alphabetic order :)
Also, it seems like the naming convection is to separate words with "_" IMO.

Reply:
Thanks for the comment.
I'll change it.

Reply:
Done. 
@ivarshav please review

Reply:
Great, thanks!